README.md

```diff
@@ -104,6 +104,23 @@ Therefore, `:job_template` option is ignored.
 set :job_template, "/bin/zsh -l -c ':job'" # ignored
 ```
 
+### Setting variables
```

Comment:
This section describes the notable differences in compatibility with Whenever.
Regarding variables, I understand that there is no difference in behavior with Whenever, so I think it is better to describe as another section. e.g.)

```
## Usage
## How it works
## Compatibility with Whenever
## Setting variables
## Development
## Contributing
```

Reply:
Or maybe it's a good idea to write it as a subsection of Usage.

Reply:
> Or maybe it's a good idea to write it as a subsection of Usage.

Thanks. I will follow it.

Reply:
> This section describes the notable differences in compatibility with Whenever.

By the way, **"Incompatibility with Whenever"** seems understandable to me than "Compatibility with Whenever" (because of saying the **difference**). 😅

Reply:
Done via 71bdcfa8b5f3b5115b2b55e9bb95a41ed55b88ec